composablejs/package.json

```diff
@@ -1,24 +1,8 @@
 {
-  "name": "@composable",
+  "name": "composable",
   "version": "0.1.0",
   "private": true,
   "license": "Public Domain",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "exports": {
-    "./types/*": [
-      "./dist/packages/types/dist/**/*.js",
-      "./dist/packages/types/dist/**/*.d.ts"
-    ],
-    "./utils/*": [
-      "./dist/packages/utils/dist/**/*.js",
-      "./dist/packages/utils/dist/**/*.d.ts"
-    ],
-    "./oracle_setup/*": [
-      "./dist/packages/oracle_setup/dist/**/*.js",
-      "./dist/packages/oracle_setup/dist/**/*.d.ts"
-    ]
-  },
   "scripts": {
     "build": "tsc -b -f tsconfig.json",
```

Comment:
```suggestion
    "build": "tsc -b -f tsconfig-base.json",
```

Reply:
Actually, the `build` command doesn't work with the refactoring but we can still compile the whole package using lerna.
So i guess we can get rid of that command. (Note: Update `Makefile` as well.)